src/site/_site/Documentation/index.js

```diff
@@ -22,13 +22,13 @@ export default class Documentation extends React.Component {
         <h3 className="Documentation_heading">Examples</h3>
         <div className="Documentation__section">{children}</div>
         <h3 className="Documentation_heading">Documentation</h3>
-        <div
-          className="Documentation__section"
-          dangerouslySetInnerHTML={HTML_DOCS}
-        />
+        <div className="Documentation__section" dangerouslySetInnerHTML={HTML_DOCS} />
       </div>
     ) : (
-      <div className={className}>{children}</div>
+      <div className={className}>
+        <h1>Bare Component Demo</h1>
```

Comment:
This h1 should be removed - we want the full page demo to be the component only.  If you need a h1 for the tests to pass, we should add an issue to figure out the right approach for those and let's get this in.